src/Nancy/Responses/DefaultXmlSerializer.cs

```diff
@@ -39,13 +40,36 @@ public void Serialize<TModel>(string contentType, TModel model, Stream outputStr
         {
             var serializer = new XmlSerializer(typeof(TModel));
 
-            if (XmlSettings.EncodingEnabled)
+            if (XmlSettings.IncludeWhiteSpace)

```

Comment:
Can't the entire `if`-block be removed completely? The only thing that is changed is the use of the `XmlWriterSettings` where only `Indent` is set based on the value of `XmlSettings.IncludeWhatSpace` ?